app/views/recipes/_recipe.html.erb

```diff
@@ -1,29 +1,28 @@
-<div id="<%= dom_id recipe %>">
+<div class="card">
+  <div class="inner-card">
+    <div>
```

Comment:
- [ ] [REQUIRED] Good job so far! Has an `index` page for `recipes` list that matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_index.png?raw=true).
   * You can remove a recipe item only if you're the owner
   * Clicking on a recipe item takes you to the recipe details page `/recipes/:id/` as shown in the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_food.png?raw=true)
   * On the recipe show page, you should be able to add ingredient.
   * On the recipe show page, you should be able to generate a shopping list that matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/general_shopping_list.png?raw=true) and url `/general_shopping_list`
   *  Has an index page for recipes list that matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_index.png?raw=true).
You can remove a recipe item only if you're the owner
Clicking on a recipe item takes you to the recipe details page /recipes/:id/ as shown in the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_food.png?raw=true)

 * On the recipe show page, you should be able to add ingredients.
 * On the recipe show page, you should be able to generate a shopping list that matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/general_shopping_list.png?raw=true) and url /general_shopping_list

However, I couldn't implement these functions because I couldn't find them. Please solve this problem 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/209192793-7585acf9-5822-45d8-840f-cbeb56324733.png)